README.md

```diff
@@ -5,41 +5,29 @@ This project is currently under active development, and as such, all source code
 
 ## General
 
-figure 1. Intel Data Center GPU Driver Container Image Releasing Flow
+**Intel Data Center GPU Driver for OpenShift project** is used to build, package, certify and release Intel data center GPU driver container images for [Red Hat OpenShift Container Platform (OCP)](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform). Users can make use of the pre-built images to provision Intel® Data Center GPU cards on OCP. They can also use [Intel data center GPU driver Dockerfile](docker/intel-dgpu-driver.Dockerfile) released from this project as a reference to on-premise build the driver container images and provision the GPU cards. Currently, [Intel® Data Center GPU Flex Series](https://www.intel.com/content/www/us/en/products/docs/discrete-gpus/data-center-gpu/flex-series/overview.html) are supported. [Intel® Data Center GPU Max Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/max-series.html) support is planned in a future release.
 
-![figure 1. Intel Data Center GPU Driver Container Image Releasing Flow](pics/releasing_flow.jpg)
+Intel data center GPU driver container image is built from [Intel GPU Driver Repository](https://github.com/intel-gpu). It includes:
+-	[Intel® Graphics (i915) Driver Backports for Linux for RHEL](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/redhat/main)
+-	[Intel® Converged Security Engine (CSE) Backports for RHEL-8.6](https://github.com/intel-gpu/intel-gpu-cse-backports/tree/rhel86)
```

Comment:
Refer link to main branch for cse and pmt. The rhel86 branch is not being maintained anymore.

Reply:
let me submit another PR to record this change.